libraries/recipe.rb

```diff
@@ -30,8 +30,12 @@ def self.my_cluster_key(node)
   end
 
   # Returns the data bag item corresponding to my cluster (not other clusters)
+  # If the cluster is a site in the multisite cluster, return data after merging with the multisite data bag item
   def self.my_cluster_data(node)
     @my_cluster_data ||= CernerSplunk::DataBag.load(node['splunk']['config']['clusters'].first, secret: node['splunk']['data_bag_secret'])
+    return @my_cluster_data if @my_cluster_data.nil?
+    @multisite_bag_data ||= CernerSplunk::DataBag.load(@my_cluster_data['multisite'], secret: node['splunk']['data_bag_secret']) || {}
+    CernerSplunk::SplunkApp.merge_hashes({ cluster_configs: @multisite_bag_data.to_h }, { cluster_configs: @my_cluster_data.to_h })[:cluster_configs]
```

Comment:
can this be `CernerSplunk::SplunkApp.merge_hashes(@multisite_bag_data.to_h, @my_cluster_data.to_h)`? Why nest things under the extra cluster_configs key?

Reply:
I don't think that is possible because from what I understand the hashes needs to have a key which will act as the app name. Key's value which is a hash gets deep merged. If I don't have a key I would expect that only those keys inside the hash whose values are hashes to get merged.

Reply:
k, I didn't realize it only merged keys at the top level that have a hash as a value.  carry on